docs/example.rst

```diff
@@ -0,0 +1,128 @@
+Example
+=======
+
+.. code::
+
+   Colour Dialogue
+   ===============
+
+   Start
+   -----
+   name: start
+
+   User dials in
+   ~~~~~~~~~~~~~
+   type: dial-event
+   name: user-dials-in
+   channels:
+      -[channel] *120*123#
+      -[channel] *120*456#
+
+
+   Favourite Colour
+   ~~~~~~~~~~~~~~~~
+   name: fav-colour
+   type: choice
+   save: colour
+
+   question[multiple-choice]:`
+     Hi {@msisdn}. What is your favourite colour?
+     1. Red {$red}
+     2. Blue {$blue}
+   `
+   routes:
+     red: chose-red
```

Comment:
Ah, I see the type is now `symbol` and is picked up automatically by the parser.